ftw/iframeblock/tests/test_builder.py

```diff
@@ -12,11 +13,22 @@ def setUp(self):
 
     @browsing
     def test_add_iframeblock(self, browser):
+        block_title = u'My iFrame block'
         content_page = create(Builder('sl content page'))
         create(Builder('iframe block')
-               .titled(u'My iFrame block')
+               .titled(block_title)
                .within(content_page))
 
         browser.login().visit(content_page)
 
-        self.assertTrue(len(browser.css('.sl-block')), 'Expect one block')
+        if IS_PLONE_5:
+            nodes = content_page.getChildNodes()
+            iframe_block_title = nodes._data[0].title
+
+            self.assertEqual(block_title, iframe_block_title,
+                             'We expect that a iframe block was created '
+                             'within the content page. Therefore the block '
+                             'must be the only child element of the content '
+                             'page having the title given.')
+        else:
+            self.assertTrue(len(browser.css('.sl-block')), 'Expect one block')
```

Comment:
Here in Plone 4.3 we find a `sl-block` in the DOM. Now in Plone 5.1 for some reason the view is never called (and not rendered). The test I implemented above having the `IS_PLONE_5` condition works and therefore shows, that the node obj exists.
Do you have any Idea why the view is never called?

Reply:
It could have a render issue? If a block fails to render it dies silently. Maybe check how it does look on a booted system.

Reply:
On the booted system it is working and also has the same css selector checked for in the test.

![image](https://user-images.githubusercontent.com/29920936/55547201-18753980-56d1-11e9-8b22-0a00d9a50ce2.png)

Reply:
This is weird...

Reply:
It seems that this was an issue of the properties which were not registered. The test as it was before works now again and the block is rendered.